docker-compose/docker-compose-certify/docker-compose.yml

```diff
@@ -94,7 +94,7 @@ services:
     networks:
       - network
   certify:
-    image: mosipdev/inji-certify:develop
+    image: mosipqa/inji-certify:0.9.x
```

Comment:
I believe it should be mosipqa/inji-certify:release-0.9.x, is it not ?

Reply:
in rancher i see this as the image name for certify and artifactory, so added the same